src/plugins/shared-variables/shared-variables.ts

```diff
@@ -26,5 +26,10 @@ export const SharedVariables = SharedModel.named("SharedVariables")
     self.addVariable(variable);
     return variable;
   },
+}))
+.views(self => ({
+  getVariables() {
+    return self.variables;
```

Comment:
What is the reason to use this instead of just having callers access self.variables directly?

Reply:
I modified the `variablesAPI` to include `getVariables()`, which is used to determine unused variables in quantity playground. Maybe there's a better way to handle this? But if there is I would like to push it for the next time I modify diagram view, since it seems like merging this work might alleviate some headaches for QA.